crates/bevy_gizmos/src/gizmos.rs

```diff
@@ -154,17 +153,17 @@ impl<'w, 's, T: GizmoConfigGroup> Gizmos<'w, 's, T> {
     /// # use bevy_render::prelude::*;
     /// # use bevy_math::prelude::*;
     /// fn system(mut gizmos: Gizmos) {
-    ///     gizmos.line_gradient(Vec3::ZERO, Vec3::X, LegacyColor::GREEN, LegacyColor::RED);
+    ///     gizmos.line_gradient(Vec3::ZERO, Vec3::X, Color::GREEN, Color::RED);
     /// }
     /// # bevy_ecs::system::assert_is_system(system);
     /// ```
     #[inline]
-    pub fn line_gradient(
+    pub fn line_gradient<C: Into<Color>>(
         &mut self,
         start: Vec3,
         end: Vec3,
-        start_color: LegacyColor,
-        end_color: LegacyColor,
+        start_color: C,
+        end_color: C,
```

Comment:
Really nice that we can explicitly say this function expects any color space, but `start` and `end` colors need to be in the same space.